en/snap/with-plugins.md

```diff
@@ -56,269 +62,308 @@ Here is the list of Content Management System plugins and extensions that are su
+| Installment Terms      | An arrangement for payment by installments\.     |
+| 3D Secure              | You must enable 3D Secure for secure card transactions\. Please contact us if you wish to disable this feature in the *Production* environment\.   |
+| Save Card              | This will allow your customer to save their card on the payment popup, for faster payment flow on future transactions. |
 
 </article>
 </details>
 
 #### Magento 2 Plugin Notification Configuration
-1. Login to your [Midtrans&nbsp;  Account](https://dashboard.midtrans.com), select your environment (sandbox/production), go to menu `settings -> configuration`
-
-| URL Role | Redirect URL|
-|----------|-------------|
+To configure the Magento 2 plugin notification URL, follow the steps given below.
+1. Login to your MAP account.
+2. In the **Environment** list, click the appropriate environment.
+3. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed. Follow the steps given below.
+   * Enter **Payment Notification URL**.
+   * Enter **Finish Redirect URL**.
+   * Enter **Error Redirect URL**.
+   * Enter **Unfinish Redirect URL**.
+   * Click **Update**.
+4. On the Home page, go to **SETTINGS > SNAP PREFERENCES > System settings**. Follow the steps given below.
+   * Enter **Finish Redirect URL**.
+   * Enter **Error Redirect URL**.
+   * Enter **Unfinish Redirect URL**.
+   * Click **Save**.
+
+A confirmation message is displayed.
+The plugin notification URL is configured successfully.
+
+The table given below shows the fields and the URL.
+
+| Field                    | URL                                       |
+| ------------------------ | ----------------------------------------- |
 | Payment Notification URL | [your-site-url]/snap/payment/notification |
-| Finish Redirect URL | [your-site-url]/snap/index/finish |
-| Error Redirect URL | [your-site-url]/snap/index/finish |
-| Unfinish Redirect URL | [your-site-url]/snap/index/finish |
+| Finish Redirect URL      | [your-site-url]/snap/index/finish         |
+| Error Redirect URL       | [your-site-url]/snap/index/finish         |
+| Unfinish Redirect URL    | [your-site-url]/snap/index/finish         |
 
-> **Note:**
->
-> Please make sure to input **http://** or **https://** when filling Notification URL and Redirect URL, according to your web-server configuration. 
-> 
-> If you are not sure, try opening your web URL in a browser, and check the URL is **http** or **https** on the address bar.
-
-2. Go to menu **Settings > Snap Preference > System Settings**
-    * Insert `[your-site-url]/snap/index/finish` link as Finish/Unfinish/Error Redirect URL.
+?> ***Note***: Based on your web-server configuration, please make sure to prefix the *Notification URL* and *Redirect URL*, with **http://** or **https://**.<br>If you are not sure, open your web URL in a browser, and check the URL is **http** or **https** on the address bar.
```

Comment:
While **you change** the appearance of the note,
> <img width="1022" alt="Screen Shot 2020-12-02 at 14 20 00" src="https://user-images.githubusercontent.com/13027142/100841165-7d39f600-34a9-11eb-9e65-a65f3be4b219.png">

Madhura **does not change** it:
> <img width="1029" alt="Screen Shot 2020-12-02 at 14 15 46" src="https://user-images.githubusercontent.com/13027142/100841102-64c9db80-34a9-11eb-9b60-deaede36a033.png">

Can we have consistency please? Decide which one looks better, and be consistent.